lib/puppet/provider/dsc_base_provider/dsc_base_provider.rb

```diff
@@ -283,9 +285,9 @@ def should_to_resource(should, context, dsc_invoke_method)
     # During a Puppet agent run, the code lives in the cache so we can use the file expansion to discover the correct folder.
     root_module_path = $LOAD_PATH.select { |path| path.match?(%r{#{resource[:dscmeta_module_name].downcase}/lib}) }.first
     resource[:vendored_modules_path] = if root_module_path.nil?
-                                         File.expand_path(Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/dsc_resources')
+                                         File.expand_path(Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/dsc_resources') # rubocop:disable Style/StringConcatenation
```

Comment:
Rubocop: Hey man! You can't do that
Lombardi: STFU
Me: LGTM

Reply:
I felt like in this case, with going up-level, it was more readable/understandable with the + instead of interpolation - looks like steps to get to what you want, imo

Reply:
Hey man, you had me at `rubocop:disable`.